Add documentation for Kiosker blackout detail binary sensor (#46281)
home-assistant/home-assistant.github.io: source/_integrations/kiosker.markdown

  - Shows whether the screensaver is currently active.
- **Blackout state**
  - Shows whether the screen is currently blacked out. Has extra attributes with the blackout data when active.
- **Blackout dismissible** (disabled by default)

>   - Shows whether the current blackout is dismissible by the user. Defaults to unknown if no visible blackout.

Reviewer: ```suggestion
  - Shows whether the current blackout is dismissible by the user. Shows as unknown when no blackout is active.
```
